### PR TITLE
Update batch submitter ethers to 5.4.0

### PR DESCRIPTION
**Pull request by karlfloersch**
**Description**
Update Ethers to version 5.4.0 -- https://github.com/ethers-io/ethers.js/releases/tag/v5.4.0 -- a version which explicitly supports eip1559

Note that even under our previous provider, our transactions are supported. See https://hackmd.io/@q8X_WM2nTfu6nuvAzqXiTQ/1559-wallets#Legacy-Transaction-Support

### CODE REVIEW

**Comment by tynes** [comment]
Unclear why the integration tests were failing here

**Comment by tynes** [comment]
This seems to cause a regression, multiple times the tests fail in the same place

**Comment by karlfloersch** [comment]
Yeah that confused me. Going to have to dig in deeper.... Right now this is a pretty low priority upgrade because we're still not making use of the new EIP1559 tx format.

**Comment by tynes** [comment]
We should see how much ETH in fees we would save as that would motivate how much priority migrating to using EIP1559 is

File: packages/batch-submitter/package.json [hunk 1]
```diff
@@ -36,8 +36,8 @@
     "@eth-optimism/contracts": "^0.4.5",
     "@eth-optimism/core-utils": "^0.5.1",
     "@eth-optimism/ynatm": "^0.2.2",
-    "@ethersproject/abstract-provider": "^5.0.5",
-    "@ethersproject/providers": "^5.0.14",
+    "@ethersproject/abstract-provider": "^5.4.0",
+    "@ethersproject/providers": "^5.4.0",
```
Comment (elenadimitrova) [inline packages/batch-submitter/package.json:40]: Should probably bump this version to 5.4.3 to accommodate [fix to forward missing EIP-1559 fields to call and estimateGas](https://github.com/ethers-io/ethers.js/releases/tag/v5.4.3)

**Pull request closed without merging**